traits/trait_types.py

```diff
@@ -2787,6 +2815,19 @@ def full_info(self, object, name, value):
 class Dict(TraitType):
     """ A trait type for a dictionary with specified key and value types.
 
+    Note that this trait type creates copies of values on assignment, rather
+    than assigning the exact instance. For example, consider::
+
+        >>> class A(HasTraits):
+        ...     x = Dict()
+        ...
+        >>> b = {}
+        >>> a = A(x=b)
+        >>> a.x
+        {}
+        >>> b['one'] = 1
+        >>> a.x
+        [1, 2, 3]
```

Comment:
Looks like copypasta here
```suggestion
        {}
```